src/main/java/apleasebot/tasks/Task.java

```diff
@@ -0,0 +1,32 @@
+package apleasebot.tasks;
+
+/**
+ * Abstract class to declare what methods Tasks should implement
+ * Also provides some methods for the children classes to inherit
+ */
+public abstract class Task {
+    protected String name;
+    protected boolean isDone;
+
+    /**
+     * Constructor for basic Task object
+     * @param name Name of task
+     * @param isDone Completion status of the task for storage and loading purposes
+     */
+    public Task(String name, boolean isDone) {
+        this.name = name;
+        this.isDone = isDone;
+    }
+
+    public void markDone() {
+        this.isDone = true;
+    }
+    public void markUndone() {
+        this.isDone = false;
+    }
+    public int done() {
+        return this.isDone ? 1 : 0;
+    };
```

Comment:
I like the intention behind this method, but perhaps can name it other than just "done"?
```suggestion
    public int checkDone() {
        return this.isDone ? 1 : 0;
    };
```